torch/fx/graph_module.py

```diff
@@ -26,6 +27,24 @@ def patched_getline(*args, **kwargs):
     return _orig_getlines(*args, **kwargs)
 linecache.getlines = patched_getline
 
+def deserialize_graphmodule(root : torch.nn.Module, src : str):
+    """
+    Deserialize a GraphModule given the original `root` module and the generated
+    `forward()` source code (`src`). This will exec() the source of the forward
+    onto the root module to create a well-formed Module with code analogous
+    to the original code. Then it symbolically traces through it to get the
+    GraphModule
+    """
+    root = copy.deepcopy(root)
+    from .symbolic_trace import symbolic_trace
+    gbls: Dict[str, Any] = {
+        'torch': torch
+    }
+    exec_with_source(src, gbls)
+    cls = type(root)
+    for k, v in gbls.items():
```

Comment:
I think it sets an attribute called `torch` on every module :) I think the same is true below too

Reply:
It also dumps `__builtins__` onto the Module for some reason